docs/src/main/asciidoc/amazon-lambda-http-guide.adoc

```diff
@@ -3,36 +3,230 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 ////
-= Quarkus - Amazon Lambda HTTP Applications 
+= Quarkus - Amazon Lambda with Resteasy, Undertow, or Vert.x Web 
 
 include::./attributes.adoc[]
 
 
 The `quarkus-amazon-lambda-http` extension allows you to write microservices with Resteasy (JAX-RS),
-Undertow (servlet), or Vert.x Web and make these microservices deployable to Amazon AWS using
-https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/what-is-sam.html[Amazon's SAM] framework.
+Undertow (servlet), or Vert.x Web and make these microservices deployable as an Amazon Lambda
+using https://aws.amazon.com/api-gateway/[Amazon's API Gateway] and https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/what-is-sam.html[Amazon's SAM framework].
+
+You can deploy your Lambda as a pure Java jar, or you can compile your project to a native image (Graal) and deploy that for a smaller
+memory footprint and startup time.
 
 == Prerequisites
 
 To complete this guide, you need:
 
-* less than 15 minutes
-* JDK 1.8 (Azure requires JDK 1.8)
+* less than 30 minutes
+* JDK 1.8 (AWS requires JDK 1.8)
 * Apache Maven 3.5.3+
 * https://aws.amazon.com[An Amazon AWS account]
-* https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-reference.html#serverless-sam-cli[Amazon SAM CLI]
+* https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html[AWS SAM CLI]
 
 == Getting Started
 
-This guide can be used on existing projects or new projects.  Whether the project is a new green field project or you're
-attempting to migrate an existing application, this guide will walk you through the steps necessary to use Quarkus powered
-applications on Amazon's infrastructure.
+This guide walks you through generating an example Java project via a maven archetype.  Later on it walks through the structure
+of the project so you can adapt any existing projects you have to use Amazon Lambda.
+
+== Installing AWS bits
+
+Installing all the AWS bits is probably the most difficult thing about this guide.  Make sure that you follow all the steps
+for installing AWS SAM CLI.
+
+== Creating the Maven Deployment Project
+
+Create the Quarkus AWS Lambda maven project using our Maven Archetype.
+
+
+[source, subs=attributes+]
+----
+mvn archetype:generate \
```

Comment:
This is fine for now as we don't have anything better, but eventually I hope we can have better create-project tooling that can be extended to handle cases like this.